SEED-platform/seed
# 3193 feat/email template help

#### Any background context you want to provide?
Organization Settings Email Tempaltes page had no instructions about using the email templates 

#### What's this PR do?
Adds instructions to the Email Templates page

#### How should this be manually tested?
Go to "+" > Email Templates > confirm there are instructions at the top of the page

#### What are the relevant tickets?
#3193

#### Screenshots (if appropriate)
![Screen Shot 2022-04-01 at 9 15 21 AM](https://user-images.githubusercontent.com/58446472/161294425-97734975-83d1-4cdc-b62e-6c826b4d4dc8.png)
